app/commands/v2/publish.rb

```diff
@@ -171,19 +174,29 @@ def update_dependencies?
         EditionDiff.new(edition).field_diff.present?
       end
 
-      def send_downstream(content_id, locale, update_type)
-        return unless downstream
-
+      def send_downstream_live
         queue = update_type == 'republish' ? DownstreamLiveWorker::LOW_QUEUE : DownstreamLiveWorker::HIGH_QUEUE
-
         DownstreamLiveWorker.perform_async_in_queue(
           queue,
+          worker_params
+        )
+      end
+
+      def send_downstream_draft
+        queue = update_type == 'republish' ? DownstreamDraftWorker::LOW_QUEUE : DownstreamDraftWorker::HIGH_QUEUE
+        DownstreamDraftWorker.perform_async_in_queue(
+          queue,
+          worker_params.merge(update_dependencies: update_dependencies?)
```

Comment:
Whereas here I think we want to have `update_dependencies` as false as if only concern is access limits we don't need to update any dependencies